packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
+docker run --name celo-accounts -d --restart always -p 8545:8545 -v $PWD/accounts:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal,istanbul
 ```
 
-### Deploy the Validator and Proxy nodes
+### Obtain and lock up some Celo Gold for staking
 
-We initialize the Docker containers for the Validator and the Proxy, building from an image for the network and initializing Celo with the genesis block found inside the Docker image:
+To participate in The Great Celo Stake Off (aka TGCSO) and get fauceted it's necessary to register online via an [online form](https://docs.google.com/forms/d/e/1FAIpQLSfbn5hTJ4UIWpN92-o2qMTUB0UnrFsL0fm97XqGe4VhhN_r5A/viewform). While you wait, let's deploy the remaining components:
 
-```bash
-docker run -v $PWD/proxy:/root/.celo $CELO_IMAGE init /celo/genesis.json
-docker run -v $PWD/validator:/root/.celo $CELO_IMAGE init /celo/genesis.json
-```
-
-{% hint style="danger" %}
-**Warning**: There is a known issue `(Fatal: Failed to read passphrase: liner: function not supported in this terminal” rather than just failing)` running geth inside Docker that happens eventually. So if that command fails, please try again, and if it continues to fail after several attempts, check this page [this page](https://forum.celo.org/t/setting-up-a-validator-faq/90).
-{% endhint %}
+### Deploy a validator
 
-In order to allow the node to sync with the network, give it the address of existing nodes in the network to the Proxy. The Validator will sync through the Proxy:
+To actually register as a validator, we'll need to generate a validating signer key. On your Validator machine (which should not be accessible from the public internet), follow very similar steps:
 
 ```bash
-docker run -v $PWD/proxy:/root/.celo --entrypoint cp $CELO_IMAGE /celo/static-nodes.json /root/.celo/
+# On the validator machine
+mkdir -p celo-data-dir/validator
+cd celo-data-dir/validator
+docker run -v $PWD/validator:/root/.celo $CELO_IMAGE init /celo/genesis.json
+docker run -v $PWD/validator:/root/.celo --entrypoint /bin/sh -it $CELO_IMAGE -c "geth account new"
+export CELO_VALIDATOR_SIGNER_ADDRESS=<YOUR-VALIDATOR-SIGNER-ADDRESS>
 ```
 
-#### Running the Proxy
-
-At this point we are ready to start up the Proxy:
+As the Valdiator account, we'll need to create a proof-of-possession of the Validator signer key with `celocli`. For that we can run the node quickly just to sign:
 
 ```bash
-docker run --name celo-proxy -d --restart always -p 8555:8545 -p 8556:8546 -p 30313:30303 -p 30313:30303/udp -p 30503:30503 -p 30503:30503/udp -v $PWD/proxy:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal,istanbul --etherbase=$CELO_PROXY_ADDRESS --proxy.proxy --proxy.proxiedvalidatoraddress $CELO_VALIDATOR_ADDRESS --proxy.internalendpoint :30503
+# On the validator machine
```

Comment:
This is unnecessary at this point, the node doesn't need to be running now that we have the new proof-of-possession command 😄